expyfun/stimuli/_tracker.py

```diff
@@ -223,6 +237,10 @@ def respond(self, correct):
                     self._n_reversals += 1
                 if self._direction <= 0:
                     self._direction = 1
+        if self._repeat_limit == 'reversal':
```

Comment:
Yeah I guess I am OK with that. Really you should not be calculating the threshold from reversals or trials if you are more than brushing up against the min every now and then.